pages/activity/entries/images.js

```diff
@@ -0,0 +1,21 @@
+import StyledReturn from "@/components/StyledReturn/StyledReturn";
+import Link from "next/link";
+import { StyledGallery } from "@/components/StyledGallery/StyledGallery";
+export default function ImageCards({ activities }) {
+  return (
+    <>
+      <StyledReturn />
+      <h1> Your Images</h1>
+      {activities.map((activity) => {
+        return (
+          <>
+            <Link href={`/activity/${activity.id}`}>
+              <p>{activity.image}</p>
+              {activity.imageURL && <StyledGallery src={activity.imageURL} />}
+            </Link>
+          </>
```

Comment:
you don't need fragments here and you are missing the key prop

```suggestion
            <Link href={`/activity/${activity.id}`} key={activity.id}>
              <p>{activity.image}</p>
              {activity.imageURL && <StyledGallery src={activity.imageURL} />}
            </Link>
```